examples/cartesian_reconstruction.py

```diff
@@ -0,0 +1,98 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI image from the sparse kspace
+measurments.
+
+Import neuroimaging data
+------------------------
+
+We use the toy datasets available in pysap, more specifically a 2D brain slice
```

Comment:
we should have somewhere the specification of the brain slice (origin, contrast, FOV, slice thickness, etc...)

Reply:
This does make sense, though I dont know whether we need to do this here, given we will be exporting the data. @philouc could give a better idea on this.

Reply:
Let's have that as an issue and resolve this.

Reply:
Yes. Resolving it here